kpconv_torch/datasets/common.py

```diff
@@ -196,22 +197,43 @@ def batch_neighbors(queries, supports, q_batches, s_batches, radius):
 class PointCloudDataset(Dataset):
     """Parent class for Point Cloud Datasets."""
 
-    def __init__(self, name):
+    def __init__(
+        self,
+        config,
+        datapath,
+        dataset,
+        chosen_log=None,
+        infered_file=None,
+        output_dir=None,
+        split="training",
+    ):
+        Dataset.__init__(self)
```

Comment:
`Dataset` is an abstract class (see https://github.com/pytorch/pytorch/blob/main/torch/utils/data/dataset.py#L41), it seems weird to call its constructor. ;-)

Reply:
Plus, we should not add code **before** the docstring.

Reply:
ok. Error of inattention...

Reply:
You still call the constructor of `Dataset`, it is not necessary in my opinion (see my first comment).